docs/nuget-org/nuget-org-faq.md

```diff
@@ -75,7 +75,7 @@ First, make sure you're using the latest versions of NuGet. If that version cont
 
 *To capture MTR:*
 
-- Download WinMTR from [http://winmtr.net/download/](http://winmtr.net/)
+- Download [WinMTR](https://sourceforge.net/projects/winmtr/files/WinMTR-v092.zip/download).
```

Comment:
I think it would be better to send the user to https://sourceforge.net/projects/winmtr/ and let them click the "download" button. They can read details about the tool and make an informed decision.

Reply:
The most obvious button there downloads 0.8 by default, which in my experience doesn't work on my machine. 0.92 is tucked away in a less obvious spot

Reply:
Just tested it.. didn't work for me either :( I guess we have no choice. I will send a message to the admins on SourceForge when I get back...